src/apps/plutus/arbitrum/plutus.stake.contract-position-fetcher.ts

```diff
@@ -0,0 +1,44 @@
+import { Inject } from '@nestjs/common';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { ContractPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { PlutusContractFactory, MasterChef } from '../contracts';
+import { PLUTUS_DEFINITION } from '../plutus.definition';
+
+import { VAULTS, ADDRESSES } from './consts';
+
+const appId = PLUTUS_DEFINITION.id;
+const groupId = PLUTUS_DEFINITION.groups.stake.id;
+const network = Network.ARBITRUM_MAINNET;
+
+@Register.ContractPositionFetcher({ appId, groupId, network })
+export class ArbitrumPlutusStakeContractPositionFetcher implements PositionFetcher<ContractPosition> {
+  constructor(
+    @Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit,
+    @Inject(PlutusContractFactory) private readonly contractFactory: PlutusContractFactory,
+  ) {}
+
+  async getPositions() {
+    return this.appToolkit.helpers.singleStakingFarmContractPositionHelper.getContractPositions<MasterChef>({
```

Comment:
Use the `MasterChefFarmContractPositionHelper` instead

Reply:
Also, perhaps rename this `MasterChef` to `PlsPlutusChef` (rename `master-chef.json` to `pls-plutus-chef.json`, then regenerate the contract factory). This will be more aligned with the naming of the other chefs you have in this integration.

Reply:
So I dont see any class or file named MasterChefFarmContractPositionHelper, are you referring to something else? maybe in the private repo?

Reply:
Re number of pools and the name, I did adjust it. But the contract itself has the name MasterChef. I think they were in a hurry to deploy and didnt have the confidence to personalize the contract, and therefore left the pool length parameters in. Either that, or its for upgradeability. I dont think there's a use case to split up plutus staking into multiple pools.

Reply:
I see, you mean the MasterChefContractPositionBalanceHelper